lib/matplotlib/tests/test_colorbar.py

```diff
@@ -465,13 +489,13 @@ def test_colorbar_log_minortick_labels():
         pcm = ax.imshow([[10000, 50000]], norm=LogNorm())
         cb = fig.colorbar(pcm)
         fig.canvas.draw()
-        lb = cb.ax.yaxis.get_ticklabels(which='both')
+        lb = [l.get_text() for l in cb.ax.yaxis.get_ticklabels(which='both')]
         expected = [r'$\mathdefault{10^{4}}$',
                     r'$\mathdefault{2\times10^{4}}$',
                     r'$\mathdefault{3\times10^{4}}$',
                     r'$\mathdefault{4\times10^{4}}$']
-        for l, exp in zip(lb, expected):
-            assert l.get_text() == exp
+        for exp in expected:
```

Comment:
This answers your question above as well:  Before we used to manually clip the ticks on the colorbar axes in the private `_ColorbarLocator` and `_ColorbarLogLocator`.  Before those, we manually added _all_ the ticks and there were no minor log ticks.  This PR is a further improvement in that we get rid of these locators, and just use whatever normal Locator the user wants.  This will allow any norm with a scale to be accurate represented on the colorbar.  

As a consequence, `cb.ax.yaxis.get_ticklabels` behaves exactly the same as for any `yaxis` and all these extra ticks are exactly what happens with normal axises.  Thats a separate battle, and should/could probably be fixed, but it should be fixed for all axis, not here.  Again the point is to remove as much special casing for colorbar axes as possible.  

A long way of saying that the full list of ticks is very long because it includes many ticks out of sight, and I don't think its necessary for a more precise test here...